addons/html_editor/static/src/core/clipboard_plugin.js

```diff
@@ -120,6 +120,8 @@ export class ClipboardPlugin extends Plugin {
     }
 
     onCut(ev) {
+        const selection = this.dependencies.selection.getEditableSelection();
+        this.dispatchTo("before_cut_or_drag_handlers", selection);
```

Comment:
I don't like this because it's "x_or_y" but I don't know yet by which singular word it could be replaced.